web/modules/custom/mymodule/src/Plugin/Field/FieldType/RealName.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Drupal\myplugin\Plugin\Field\FieldType;
+
+use Drupal\Core\Field\FieldItemBase;
+use Drupal\Core\TypedData\DataDefinition;
+use Drupal\Core\Field\FieldStorageDefinitionInterface as StorageDefinition;
+
+/**
+ * Plugin implementation of the 'Real Name' field type.
+ *
+ * @FieldType(
+ *   id = "real_name",
+ *   label = @Translation("Real Name"),
+ *   description = @Translation("Stores the first name and last name"),
+ *   default_widget = "RealNameDefaultWidget",
+ *   default_formatter = "RealNameDefaultFormatter"
+ * )
+ */
+class RealName extends FieldItemBase {
+
+  /**
+   * Field type properties definition.
+   */
+  public static function propertyDefinitions(StorageDefinition $storage) {
+
+    $properties = [];
+
+    $properties['first_name'] = DataDefinition::create('string')
+      ->setLabel(t('First name'));
+
+    $properties['last_name'] = DataDefinition::create('string')
+      ->setLabel(t('Last name'));
```

Comment:
@anitha-innoraft - in class `t` method should be replaced by `$this->t()`

Reply:
Used t(). As getting error.